lib/grape_entity/exposure/base.rb

```diff
@@ -82,7 +83,10 @@ def serializable_value(entity, options)
         end
 
         def valid_value(entity, options)
-          value(entity, options) if valid?(entity)
+          return unless valid?(entity)
+
+          output = value(entity, options)
+          output.blank? && @default_value.present? ? @default_value : output
```

Comment:
I am not a maintainer, but I stumbled across this PR while I was looking for a sane default ability for entities.
`blank?` and `present?` are Rails specific, so to keep Grape framework agnostic you might want to shuffle this around. In fact, the `blank?` check replaces some valid options like `false` and `[]` with the default value, which I would not expect.

```suggestion
output.nil? ? @default_value : output
```
I'm not sure the `.present?` check bought you anything anyway if we keep it strictly to a nil check.

Reply:
`.blank?` and `.present?` are part of ActiveSupport and this is included in the Grap gemspec, so it can be used without adding additional dependencies

but suggest to move the getting of the value into the same named method → [`value()`](https://github.com/ruby-grape/grape-entity/blob/master/lib/grape_entity/exposure/base.rb#L64)

Reply:
@LeFnord not sure if I get this part: 
> but suggest to move the getting of the value into the same named method → value()

Reply:
@LeFnord Do you still consider the behavior of `blank?` to be appropriate though? If I had a boolean attribute with a default of `true`, then the above implementation will always return `true` no matter what, as:
```ruby
false.blank?
# => true
```

It seems to me like the only sane value to override with a default would be `nil`